src/canopy_calcs.F90

```diff
@@ -22,11 +22,22 @@ SUBROUTINE canopy_calcs
     use canopy_phot_mod
     use canopy_eddy_mod
     use canopy_bioemi_mod
+    !!use canopy_bioparm_mod, ONLY: ANEW, AGRO, AMAT, AOLD
 
     IMPLICIT NONE
 
     !Local variables
     integer i,j,k,loc
+    ! Empirical coeff.'s for Leaf Age factor calculations (see
+    ! canopy_bioparm_mod or call canopy_biop)
+    REAL(rk) :: anew
+    REAL(rk) :: agro
+    REAL(rk) :: amat
+    REAL(rk) :: aold
+    REAL(rk) :: plai          ! Past LAI [cm2/cm2]
+    REAL(rk) :: clai          ! Current LAI [cm2/cm2]
+    REAL(rk) :: ndays         !Number of days between the past and current LAI
+    REAL(rk) :: tabovecanopy, temp2 !(modlays) ! Above Canopy Temp (assigned = TEMP2 )
```

Comment:
You don't need to assign a new variable ```temp2``` here.  We already have the ```tmp2mref``` variable to use directly.  Please revise.